mod/missions/views/default/user/mission-candidate.php

```diff
@@ -60,11 +60,21 @@
 
 $skill_set = '';
 $count = 1;
+//Limiting how many skills first show up in the array
+$limit_skills = array_slice($user_skills, 0, 4);
+foreach($limit_skills as $skill) {
+    
+	$skill_set .= '<span name="user-skill-' . $count . '" class="mission-skills cut-skill" title="'.$skill->title.'">' . $skill->title . '</span>';
+	$count++;
```

Comment:
Wouldn't the count be up to 4 greater than the actual size since you're counting the first 4 twice?